src/Goerli-DssSpell.t.base.sol

```diff
@@ -123,8 +100,10 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
     SpellValues  spellValues;
 
     Hevm hevm;
-    Rates     rates = new Rates();
-    Addresses addr  = new Addresses();
+
+    Rates         rates = new Rates();
+    Addresses     addr  = new Addresses();
```

Comment:
minor nit but the alignment with these three lines triggers me

Reply:
me too, but i was following the pattern we have here: https://github.com/makerdao/spells-goerli/blob/master/src/Goerli-DssSpell.t.base.sol#L140-L164

Reply:
unless you were talking about the addr being aligned with rates and having an extra space, in which case that is fixed now :-D